app/controllers/bike_share_app.rb

```diff
@@ -1,3 +1,122 @@
 class BikeShareApp < Sinatra::Base
+  get '/' do
+    erb :index
+  end
 
+  get '/dashboard' do
+    erb :dashboard
+  end
+
+  get '/stations-dashboard' do
+    erb :"stations/dashboard"
+  end
+
+  get '/stations' do
+    @stations = Station.order(:id).paginate(:page => params[:page], :per_page => 30)
+    erb :'stations/index'
+  end
+
+  get '/stations/new' do
+    erb :"stations/new"
+  end
+
+  get '/stations/:id/detailed' do |id|
```

Comment:
this is very similar to the Station show method below. this route does not follow RESTful conventions - the one on line 33 does. if i remember correctly, y'all decided not to use this one?